### PR TITLE
added shortcut for grabbing focus of task-entry

### PR DESCRIPTION
**Pull request by VivekThazhathattil**
Hi! This is a fix for the issue: "Keyboard shortcut to put focus on bottom text box. #204".
(btw I'm not too familiar with making pull requests, so if I'm making some mistakes in the procedure, please do guide me!)

### CODE REVIEW

File: src/gtimelog/shortcuts.ui [hunk 1]
```diff
@@ -101,6 +101,13 @@
                 <property name="title" translatable="yes" context="shortcut window">Menu</property>
               </object>
             </child>
+            <child>
+              <object class="GtkShortcutsShortcut">
+                <property name="visible">True</property>
+                <property name="accelerator">&lt;primary&gt;L</property>
+                <property name="title" translatable="yes" context="shortcut window">Add new entry</property>
```
Comment (mgedmin) [inline src/gtimelog/shortcuts.ui:108]: Perhaps this should say "Focus the task entry"?  "Add" sort of implies this keybinding would trigger the Add button.

**Review by mgedmin — DISMISSED** [review]
LGTM!

**Comment by mgedmin** [comment]
I would also like a changelog entry in CHANGES.rst.  Something like

```
- Focus the task entry on Ctrl+L (GH: #213).
```

is fine

**Comment by VivekThazhathattil** [comment]
@mgedmin I added those changes..

**Review by mgedmin — APPROVED** [review]

**Pull request merged**